source/opt/log.h

```diff
@@ -39,6 +39,21 @@
 #define SPIRV_ASSERT(consumer, ...)
 #endif
 
+// Logs an error message to consumer. Accepts the following formats:
```

Comment:
(I know I haven't complained about the SPIRV_DEBUG macros, but the same criticism could apply there too.)